splice/web/api/campaign.py

```diff
@@ -89,7 +89,11 @@ def get(self):
 
         Takes an optional account_id as a query string argument.
         """
-        campaigns = get_campaigns(request.args.get('account_id'))
+        parser = reqparse.RequestParser()
```

Comment:
better create one single RP instance in `__init__` and reuse it. i know premature optimization is the root of evil :)